.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,7 +11,7 @@
   - [ ] The new extension is used in hundreds of repositories on GitHub.com
     - Search results for each extension:
       <!-- Replace FOOBAR with the new extension, and KEYWORDS with keywords unique to the language. Repeat for each extension added. -->
-      - https://github.com/search?utf8=%E2%9C%93&type=Code&ref=searchresults&q=extension%3AFOOBAR+KEYWORDS+NOT+nothack
+      - https://github.com/search?type=code&q=NOT+is%3Afork+path%3A*.FOOBAR+KEYWORDS
```

Comment:
Shouldn't this be `path%3A**/*.FOOBAR`? `path:*.FOOBAR` will only match `.FOOBAR` files in the root directory of a project, IIRC.

Reply:
Oh, right. Guess I've been spending too much time in the shell recently… 😅